test/src/test/java/org/corfudb/infrastructure/LayoutBuilderTest.java

```diff
@@ -23,6 +25,37 @@
     final int SERVER_ENTRY_3 = 3;
     final int SERVER_ENTRY_4 = 4;
 
+    @Test
+    public void emptyLayoutBuilder() {
+        final long epoch = 2;
+        UUID clusterId = UUID.randomUUID();
```

Comment:
Don't use randomUUID in tests.

Reply:
Why not ?

Reply:
Unit tests should be deterministic, whenever possible. See [#1, rule 2] (http://jasonpolites.github.io/tao-of-testing/ch4-1.1.html) and [#2] (https://martinfowler.com/articles/nonDeterminism.html).

Reply:
Only the "data" generation is random, the logic is deterministic.

Reply:
seems like a convention/rule, changed.